### DISCUSSION TITLE
zh/introduction/

### DISCUSSION THREAD

**Discussion by giscus** [ryan4yin/nixos-and-flakes-book#139] [Announcements]
# zh/introduction/

一份非官方的新手指南

https://nixos-and-flakes.thiscute.world/zh/introduction/

**Comment by nowhereman1999** [comment]
好好好，感觉Nix就是为我这种天生爱折腾各种配置环境的人的最终解决方案

**Reply by 0zhangchibang0** [reply]
还有我这种电脑多且经常换的人

**Comment by BeyondtheApex** [comment]
# 谢谢
![g](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQM7Pjh_9PuIoYLuZgeh0legCue9HSnsNHYiH0PZRXUcA&s)

**Comment by sachinchaudhary1808** [comment]
bro has less comments no...........

**Comment by WeepingDogel** [comment]
qwq
~~到此一游~~

**Reply by WeepingDogel** [reply]
汪汪

**Comment by Extious** [comment]
从 Arch 叛逃过来的有没有

**Reply by WeepingDogel** [reply]
~~有 我我我我~~
